tests/test_webpack.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2020 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test Webpack module."""
+
+import pytest
+from flask_webpackext import WebpackBundle
+from pywebpack import UnsupportedExtensionError
+
+from invenio_assets.webpack import UniqueJinjaManifestEntry, \
+    UniqueJinjaManifestLoader, WebpackThemeBundle
+
+
+def test_webpack_theme_bundle(app):
+    """Test WebpackThemeBundle."""
+    themes = {
+        'bootstrap3': dict(
+            entry={
+                'theme': './theme-bootstrap3.css',
+            }
+        ),
+        'semanticui': dict(
+            entry={
+                'theme': './theme-semanticui.css',
+            }
+        ),
+    }
+
+    # Test that bundles are created for each theme.
+    bundle = WebpackThemeBundle(
+        'tests', 'assets', default='semanticui', themes=themes)
+    assert isinstance(bundle.themes['bootstrap3'], WebpackBundle)
+    assert isinstance(bundle.themes['semanticui'], WebpackBundle)
+
+    # Test that default theme is used.
+    with app.app_context():
+        assert bundle._active_theme_bundle == bundle.themes['semanticui']
+        assert bundle.path == bundle.themes['semanticui'].path
+        assert bundle.entry == bundle.themes['semanticui'].entry
+        assert bundle.dependencies == bundle.themes['semanticui'].dependencies
+
+    # Test that APP_THEME overrides default theme.
+    app.config['APP_THEME'] = ['bootstrap3']
+    with app.app_context():
+        assert bundle._active_theme_bundle == bundle.themes['bootstrap3']
+        assert bundle.path == bundle.themes['bootstrap3'].path
+        assert bundle.entry == bundle.themes['bootstrap3'].entry
+        assert bundle.dependencies == bundle.themes['bootstrap3'].dependencies
+
+    # Test that an invalid APP_THEME
+    app.config['APP_THEME'] = ['invalid']
+    with app.app_context():
+        assert bundle._active_theme_bundle is None
```

Comment:
@slint I assume this is desired behaviour? However, I'm not sure what will happen when you try to run the webpack build project with a bundle where ``bundle.entries`` will raise an error.

Reply:
I think there are three choices:

- Let Flask-WebpackExt complain/raise when it comes across no bundle
- Raise an exception in `_active_theme_bundle` if no bundle can be determined
- Always return the `default` bundle (I don't like this one since that would mean you get Bootstrap even when you don't want it)

Reply:
Let's merge and figure out what's the behaviour :-)

Reply:
Logged in #116